plugin/plugin.go

```diff
@@ -23,6 +26,26 @@ func New(policy string) validator.Plugin {
 
 func (p *plugin) Validate(ctx context.Context, req *validator.Request) error {
 
+	log.SetFormatter(&log.JSONFormatter{})
+	log.SetOutput(os.Stdout)
+
+	promotion := Promotion{
+		Build:       req.Build.Parent,
+		Environment: req.Build.Deploy,
```

Comment:
I don't see any reason to change this to `Environment`, seems like it should use the same terminology as the underlying type.

Reply:
Yeah, it makes sense.